app/views/thinkers/_form.html.erb

```diff
@@ -19,11 +19,16 @@
   </p>
  
   <p>
-     <%= f.email_field :email %>
+     <%= f.text_field :topic_list %>
   </p>
 
   <p>
-     <%= f.text_field :topic_list %>
+    <%= f.date_field :life_date_birth, class: 'form-control', autofocus: true, placeholder: '1850' %>
```

Comment:
I realise the `date_field` helper isn't want we want: try it in Chrome, and you get an actual date picker!

We likely want to use either `number_field` (which gives you a traditional number spinner in Chrome), or a simple `text_field`.